lib/web/notifications_test.go

```diff
@@ -0,0 +1,284 @@
+package web
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/gravitational/teleport/api/types"
```

Comment:
Sorry for the delay, I will review this tomorrow. Just wanted to point you that you can fix this lint error by running `make fix-imports`

it's a quite cryptic error and it took me forever to know what it meant. it just means the imports are out of order basically.